### PR TITLE
Update to snapshot 6602655.

### PR DESCRIPTION
**Pull request by nickbutcher**

### CODE REVIEW

File: JetNews/app/build.gradle [hunk 1]
```diff
@@ -41,15 +42,18 @@ android {
     }
 
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = '1.8'
+        apiVersion = '1.3'
```
Comment (chrisbanes) [inline JetNews/app/build.gradle:46]: Add a comment

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt [hunk 2]
```diff
@@ -96,20 +97,21 @@ val paragraphsPost1 = listOf(
         listOf(Markup(MarkupType.Code, 28, 43))
     ),
     Paragraph(
-        ParagraphType.CodeBlock, "include ':app'\n" +
-                "include ':module1'\n" +
-                "include ':module1'\n" +
-                "include ':module1'\n" +
-                "include ':module1'"
+        ParagraphType.CodeBlock,
+        "include ':app'\n" +
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:101]: Nit: use raw string?

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt [hunk 3]
```diff
@@ -96,20 +97,21 @@ val paragraphsPost1 = listOf(
...
     ),
     Paragraph(
         ParagraphType.CodeBlock,
         "// Set a custom path for the four features modules.\n" +
-                "// This avoid to have an empty \"features\" module in  Android Studio.\n" +
-                "project(\":module1\").projectDir=new File(rootDir, \"features/module1\")\n" +
-                "project(\":module2\").projectDir=new File(rootDir, \"features/module2\")\n" +
-                "project(\":module3\").projectDir=new File(rootDir, \"features/module3\")\n" +
-                "project(\":module4\").projectDir=new File(rootDir, \"features/module4\")"
+            "// This avoid to have an empty \"features\" module in  Android Studio.\n" +
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:110]: Ditto

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt [hunk 4]
```diff
@@ -312,7 +314,7 @@ val paragraphsPost2 = listOf(
     Paragraph(
         ParagraphType.CodeBlock,
         "@Inject @MinimumBalance lateinit var minimumBalance: BigDecimal \n" +
-                "// @MinimumBalance is ignored!",
+            "// @MinimumBalance is ignored!",
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:317]: ditto

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt [hunk 5]
```diff
@@ -401,10 +403,10 @@ val paragraphsPost2 = listOf(
     Paragraph(
         ParagraphType.CodeBlock,
         "class MyVMFactory @Inject constructor(\n" +
-                "  private val vmMap: Map<String, @JvmSuppressWildcards Provider<ViewModel>>\n" +
-                ") { \n" +
-                "    ... \n" +
-                "}",
+            "  private val vmMap: Map<String, @JvmSuppressWildcards Provider<ViewModel>>\n" +
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:406]: Bit more ditto

File: JetNews/app/src/main/java/com/example/jetnews/ui/SwipeToRefresh.kt [hunk 6]
```diff
@@ -123,15 +122,18 @@ internal fun <T> StateDraggable(
...
     val currentValue = anchorsToState.firstOrNull { it.second == state }!!.first
     val flingConfig =
-        AnchorsFlingConfig(anchors, animationBuilder, onAnimationEnd = { reason, finalValue, _ ->
-            if (reason != AnimationEndReason.Interrupted) {
-                val newState = anchorsToState.firstOrNull { it.first == finalValue }?.second
-                if (newState != null && newState != state) {
-                    onStateChange(newState)
-                    forceAnimationCheck.value = !forceAnimationCheck.value
+        AnchorsFlingConfig(
+            anchors, animationBuilder,
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/ui/SwipeToRefresh.kt:126]: move to new line

File: JetNews/app/build.gradle (hunk 1 above)
Comment (nickbutcher) [reply JetNews/app/build.gradle:46]: Done

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt (hunk 2 above)
Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:101]: Done

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt (hunk 3 above)
Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:110]: Done.

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt (hunk 4 above)
Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:317]: Done.

File: JetNews/app/src/main/java/com/example/jetnews/ui/SwipeToRefresh.kt (hunk 6 above)
Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/ui/SwipeToRefresh.kt:126]: Bit more done.

File: JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt (hunk 5 above)
Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/data/posts/impl/PostsData.kt:406]: Bit more done.

**Pull request merged**